### PR TITLE
Adding in In Library Use ReCAP requests for SCSB items

### LINKED ISSUE #792: SCSB In Library Use Should be Requestable to Firestone
https://catalog.princeton.edu/requests/SCSB-8953469?source=pulsearch

Items That are SCBS in library only should be able to be requested instead of just "Help Me Get It"

- [ ] new Type recap_in_library (Check out here:  https://github.com/pulibrary/requests/blob/master/app/models/requests/router.rb#L99)
- [ ] New Form Option
  Need to know what verbiage we will use on the page
  Some thing like "Available for In Library Use.  Please Request the item.  Once you receive a confirmation it has made it to Firestone, please book a Study Browse appointment(insert link) to see you item."  Use Recap brief message about how long the item takes to get from recap to Firestone.
- [ ] Confirmation message goes to the user similar to recap, but with Study browse link too.
- [ ] Does the same backend as Recap pick-up

### PR DESCRIPTION
**Pull request by carolyncole**
fixes #792

### CODE REVIEW

**Review by kevinreiss — CHANGES_REQUESTED** [review]
Just a couple language changes.

File: app/views/requests/request_mailer/recap_in_library_confirmation.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <p class="body-content" style="margin-bottom:0;">
+      Once you recieve the confirmation that your book is available, please utilize our study-browse to make an appointment to come in and view the item in the Library.
+    </p>
+  </td>
+</tr>
+
+<tr class="pikcup-info">
+  <td colspan="2" style="padding: 18px;">
+  <div style="text-align:center;margin:18px 0;">
+    <p class="body-content">If you are an undergraduate requesting to view books at a Library, please be sure that you have access to the building. Access can be requested through your academic department.</p>
```
Comment (kevinreiss) [inline app/views/requests/request_mailer/recap_in_library_confirmation.html.erb:29]: Rewrite as ```If you are requesting to view books at a Library, please be sure that you have access to the building. If you are unsure if your card will be active for a building you plan to visit access can be requested through your academic department.```

File: app/views/requests/request_mailer/recap_in_library_confirmation.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <% else %>
+      <%= I18n.t('requests.recap.email_conf_msg') %>
+      <% end %>
+    </p>
+  </td>
+</tr>
+<tr>
+  <td colspan="2" style="padding: 18px;">
+    <p class="body-content" style="margin-bottom:0;">
+      Once you recieve the confirmation that your book is available, please utilize our study-browse to make an appointment to come in and view the item in the Library.
```
Comment (kevinreiss) [inline app/views/requests/request_mailer/recap_in_library_confirmation.html.erb:21]: Rewrite as: ```Once you receive confirmation via email that your book is available, please utilize our study-browse service to make an appointment to come in to retrieve the item from the Library's circulation desk and view it in the Library.```

Comment (carolyncole) [reply app/views/requests/request_mailer/recap_in_library_confirmation.html.erb:21]: That seems like a really long single sentence.  How about `Once you receive confirmation via email that your book is available, please utilize our study-browse service to make an appointment.  During your appointment you will retrieve the item from the Library's circulation desk and view it in the Library.`

**Comment by carolyncole** [comment]
@kevinreiss This has been updated.  As long as you are OK with my change to the very long sentence.

**Review by kevinreiss — APPROVED** [review]
👍

**Pull request merged**